### ISSUE TITLE
Consider the VRGP entry point in the URL

### ISSUE THREAD

**Issue by bsilverajan** [aboamare/vrgp-specifications#2]
Currently the URI provided in the document is of the form

wss://[host]/mmsi

When performing any HTTP upgrade handshakes, would it be appropriate to define an entry point defined in the URI path, such as the following?

wss://[host]/[vgrp-entrypoint]/mmsi

once the origin server is contacted at this URI, it would be able to perform the HTTP upgrade handshake required for establishing the websocket connection with a GET request. Discovery can also be made using .well-known URIs, by making a request, for example to .well-known/vgrp, and obtaining a response before negotiation for the websocket connection upgrade

**Comment by lokkilok** [comment]
Yes, it's a useful thing to point out. The MCP/IALA define a full blown Service Registry which could/would be used to discover e.g. a/the "piloting" service in a certain geographic area.
